sim/src/main/cc/midasexamples/TerminationModule.h

```diff
@@ -0,0 +1,52 @@
+//See LICENSE for license details.
+
+#include "simif_peek_poke.h"
+#include "bridges/termination.h"
+#include <stdlib.h>
+#include <time.h>
+#include <iostream>
+
+class TerminationModule_t: public simif_peek_poke_t
+{
+public:
+    termination_t * terminator;
+    TerminationModule_t(int argc, char** argv) {
+        TERMINATIONBRIDGEMODULE_0_substruct_create;
+        std::vector<std::string> args(argv + 1, argv + argc);
+        terminator = new termination_t(
+          this, args,
+          TERMINATIONBRIDGEMODULE_0_substruct,
+          TERMINATIONBRIDGEMODULE_0_message_count,
+          TERMINATIONBRIDGEMODULE_0_message_type,
+          TERMINATIONBRIDGEMODULE_0_message);
+    };
+    void run() {
+        poke(reset, 1);
+				srand(time(NULL));
```

Comment:
simif_peek_poke_t provides some barebones randomization utilities, use those so that seeding is done consistently and reproducibly.

Reply:
Got it. Done.

Reply:
Done.